chart/templates/uds-package.yaml

```
@@ -9,7 +9,7 @@ spec:
     - name: Mattermost Login
       clientId: uds-swf-mattermost
       redirectUris:
-        - "https://chat.uds.dev/*"
+        - "https://chat.{{ .Values.domain }}/*"
```

```suggestion
        - "https://{{ .Values.subdomain }}.{{ .Values.domain }}/*"
```

Technically this is configurable currently - see the expose entry below.